eland/operations.py

```diff
@@ -1211,6 +1211,36 @@ def to_csv(
         df = self._es_results(query_compiler, show_progress)
         return df.to_csv(**kwargs)  # type: ignore[no-any-return]
 
+    def search_yield_pandas_dataframes(
```

Comment:
A note for future contributions, to keep things easier to understand we like to name things on both `QueryCompiler` and `Operations` the same if they do the same thing/forward downwards to `Operations` like this example. I also pluralized `dataframes` since there's more than one potentially.